### ISSUE TITLE
Reduce Api Calls

### ISSUE THREAD

**Issue by MiltonAkash** [kenny-hibino/react-places-autocomplete#174]
Google autocomplete gives us placeId, Latitude &Longitude on selecting the suggestion itself. But in this package we are manually making another call to geocode the address.

**Comment by daan-nguyen** [comment]
There is a `placeId` passed back in the [`onSelect` handler](https://github.com/kenny-hibino/react-places-autocomplete#onSelect), but I don't see long/lats anywhere in the `suggestion` object though. Are you able to point out where/when you get it during auto complete?

**Comment by IngusSkaistkalns** [comment]
+1 Google has kicked in with new billing policy, this one becomes pricy now :)

**Comment by abobykin** [comment]
@hibiken Hello! Any updates on that? )

**Comment by m-nathani** [comment]
i completely agree, it is making api call twice for geocoding, whereas place api does the geocoding implicitly now ..

**Comment by m-nathani** [comment]
ill try to create a fix and send PR if its works..

**Comment by sambokai** [comment]
@m-nathani Can you elaborate? Which request is it doing twice and how could this be solved?

**Comment by StephaneP** [comment]
based on googles documentation to group requests, you need to specify a session_id, so it groups up the calls
